### PR TITLE
Test change status

### PR DESCRIPTION
**Pull request by ghost**
## Description
Test change status post.
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
  NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: tests/Browser/Pages/Backend/Post/TestChangeStatus.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            'status' => Post::DISABLE
+        ]);
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/admin/post')
+                    ->click('table tr td a')
+                    ->pause(1000);
+            $this->assertDatabaseHas('posts',['status' => Post::ENABLE]);
+        });
+    }
+}
```
Comment (at-thaitran) [inline tests/Browser/Pages/Backend/Post/TestChangeStatus.php]: update branch

File: tests/Browser/Pages/Backend/Post/TestChangeStatus.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+class TestChangeStatus extends DuskTestCase
+{
+    use DatabaseMigrations;
+    /**
+     * test if change post status work.
+     *
+     * @return void
+     */
+    public function testChangePostStatus()
```
Comment (at-thinhnguyen) [inline tests/Browser/Pages/Backend/Post/TestChangeStatus.php]: thêm case lỗi nữa

**Comment by at-thaitran** [comment]
LGTM

**Comment by at-thinhnguyen** [comment]
sửa lại title cho đủ nhá, update status cho cái gì?

**Pull request merged**